doc/source/getting-started.rst

```diff
@@ -79,6 +79,26 @@ multiple branches at once, you can use a regular expression:
 In this example, pull requests sent to a branch whose name matches ``stable/*``
 will only require one review approval and will still require Travis to pass.
 
+You also can automatically merge PR as soon all your required status checks
```

Comment:
You can automatically merge a pull request as soon as all the required status checks pass: